streaming/flink/vpc.tf

```diff
@@ -0,0 +1,102 @@
+#---------------------------------------------------------------
+# Supporting Network Resources
+#---------------------------------------------------------------
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 3.0"
```

Comment:
`4.0` module is available and consolidates some of the code. See https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/1546

Reply:
updated